### PR TITLE
Test 'ncursesw' package

### PR DESCRIPTION
**Pull request by tstack**
* I've followed [this guide](https://docs.hunter.sh/en/latest/creating-new/create/cmake.html)
  step by step carefully. **Yes**

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed":

  * https://travis-ci.org/tstack/hunter/builds/553429479

* My change will resolve this "New package" request: https://github.com/ruslo/hunter/issues/1739

---

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -76,19 +76,19 @@ matrix:
...
         TOOLCHAIN=osx-10-13-cxx14
-        PROJECT_DIR=examples/foo
+        PROJECT_DIR=examples/ncursesw
 
     - os: osx
       osx_image: xcode9.4
       env: >
         TOOLCHAIN=ios-nocodesign-11-4-dep-9-3
-        PROJECT_DIR=examples/foo
+        PROJECT_DIR=examples/ncursesw
```
Comment (ruslo) [inline .travis.yml]: >  Here is the links to the Travis/AppVeyor with status "All passed":

iOS toolchain is failing:
- https://travis-ci.org/tstack/hunter/jobs/553429491

Please exclude it. Exclude other failing toolchains too.

**Review by ruslo — CHANGES_REQUESTED** [review]

**Comment by tstack** [comment]
Here's an updated build on my fork with the broken toolchains commented out:

  * https://travis-ci.org/tstack/hunter/builds/554072871

**Comment by ruslo** [comment]
> Here's an updated build on my fork with the broken toolchains commented out:

Thanks.

If there is no working toolchain in `appveyor.yml`, please exclude all toolchains there and add `dummy` one, example:
- https://github.com/ingenue/hunter/blob/b52b18e7ac51cec76c63a61dd81195c5bfc2a160/appveyor.yml#L35-L41

**Comment by tstack** [comment]
Added the dummy toolchain in appveyor :

  * https://ci.appveyor.com/project/tstack/hunter/builds/25752024

**Comment by tstack** [comment]
(Accidentally hit the close button)

**Review by ruslo — CHANGES_REQUESTED** [review]
> Added the dummy toolchain in appveyor 

Please add it to this pull request.

**Comment by tstack** [comment]
Sorry, didn't push the pr branch.

**Review by ruslo — APPROVED** [review]

**Pull request merged**